Resources/Prototypes/Catalog/VendingMachines/Inventories/janidrobe.yml

```diff
@@ -11,7 +11,10 @@
     ClothingOuterWinterJani: 2
     ClothingNeckScarfStripedPurple: 3
     ClothingShoesBootsWinterJani: 2 #Delta V: Add departmental winter boots
-
-  emaggedInventory:
-    ClothingUniformJumpskirtJanimaid: 2
+    ClothingUniformJumpskirtJanimaid: 2 # Delta V: the maiden outfits no longer require emagging.
     ClothingUniformJumpskirtJanimaidmini: 1
+  
+  emaggedInventory:
+    ClothingHandsTacticalMaidGloves: 1
+    ClothingUniformJumpskirtTacticalMaid: 1
+    ClothingHeadHatTacticalMaidHeadband: 1
```

Comment:
shouldn't all of this have a comment at the end of it, since it's delta exclusive?

Reply:
RIGHT, GODDAMIT, I DIDNT FORGOT THE FIRST PART BUT THIS ONE I DID :elp: